File: codekata/mathematics/sum_of_arithmetic_series.py
# sum of arithmetic series 
# get three inputs from user in a single line
a,d,n=list(map(int,input().split()))
# define arithmetic function
def arithmetic():
# to perform arithmetic operation
  ap=n/2*(2*a+(n-1)*d)
  r=round(ap)
  return r
# call arithmetic function and store it in result
result=arithmetic()
# print the result value
print(result)

File: codekata/javascript/mathematics/sum_of_Arithmetic_series.js
// Given 3 numbers A,B,C find the sum of Arithmetic Series with a=A, d=B and n=C
const readline=require('readline');
const inp=readline.createInterface({
    input:process.stdin
});
const userInput=[];
inp.on("line",(data)=>{
    userInput.push(data);
});
inp.on("close",()=>{
    var S=userInput[0].split(" ");
    var a=S[0];
    var d=S[1];
    var n=S[2];
    var ap=parseInt(n)/2*(2*parseInt(a)+(parseInt(n)-1)*parseInt(d));
    console.log(ap);
  });
